assets/app/view/game/round/auction.rb

```diff
@@ -170,6 +170,14 @@ def render_input(company)
               ]
             end
 
+          if @step.may_reduce?(company)
+            company_actions << h(:button, { on: { click: lambda {
+                                                           process_action(
```

Comment:
blame rubocop for this formatting

Reply:
put the hash on a new line

h(:button, {
  on: ...
})

Reply:
57ed230f